cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -277,3 +277,22 @@ def _clear_ftp():
                     dt.total_seconds() >= 0.0,
                     "Negative gap between log files ({}, {})".format(
                         datetimes[i][2], datetimes[i + 1][2]))
+
+
+def test_collect_unconfigured(request, soft_assert):
+    """ Test checking is collect button enable and disable after log depot was configured
+
+    """
+    configure.ServerLogDepot.Credentials.clear()
```

Comment:
You don't have to clear the credentials in the beginning. Instead, make sure that `request.addfinalizer(configure.ServerLogDepot.Credentials.clear)` is added right after `log_credentials.update(validate=False)`. Since the other tests in this module also clear the credentials in their finalizers, this should be sufficient.